src/common/graphHelpers.js

```diff
@@ -2,8 +2,8 @@ const convertGraphData = () => {
   // todo: convert passed params to consumable chart data
 
   return [
-    { x: 'May 25', y: 1 },
-    { x: 'May 26', y: 1 },
+    { x: 'May 25', y: 30, label: '30 Cores on May 25' },
+    { x: 'May 26', y: 60, label: '60 Cores on May 26 \r\n +30 from previous day' },
```

Comment:
request: need to change the word `cores` in lines 5 and 6 to `sockets`

Reply:
can do